scripts/start.sh

```diff
@@ -30,5 +30,4 @@ show_logs_temporarily() {
 show_logs_temporarily zkwasm-mongodb
 
 # Finally, attach to other services
-docker compose up prover-dry-run-service
-docker compose up prover-node
\ No newline at end of file
+docker compose up
```

Comment:
I can see after we finish mongodb init, we still have about 10-20 seconds stucking at 
"Attaching to prover-node-docker-prover-node-1" before the prover log begin rolling. 
![image](https://github.com/user-attachments/assets/f503c510-e9ca-43a2-8b56-302daebdbb72)

Thus I suggest we do the same thing for prover-dry-run-service.